target/linux/rtl838x/Makefile

```diff
@@ -0,0 +1,22 @@
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+include $(TOPDIR)/rules.mk
+
+ARCH:=mips
+CPU_TYPE:=4kec
+BOARD:=rtl838x
+BOARDNAME:=Realtek MIPS
+FEATURES:=ramdisk squashfs
+
+KERNEL_PATCHVER:=5.4
+
+define Target/Description
+	Build firmware images for Realtek RTL838x based boards.
+endef
+
+include $(INCLUDE_DIR)/target.mk
+
+DEFAULT_PACKAGES += swconfig uboot-envtools ethtool kmod-gpio-button-hotplug
```

Comment:
Do you use swconfig or DSA or both?

Reply:
Only DSA. blogic is working on proper netifd support. So probably I should remove swconfig. IIRC Lucy then does not show anything on the switch-ports.

Reply:
If you only use DSA, you should remove swconfig

Reply:
The swconfig needs to be there. There seem to be scripts during boot which check for swconfig this, and if not there, then /etc/config/network is not properly generated.

Reply:
We already have DSA-only targets like ramips/mt7621 where swconfig is removed entirely; I rather assume that your 02_network switch scripts implicitly depends on swconfig somewhere.

We should sort this out eventually, however I would not give it the highest importance at the moment.